databricks/koalas/indexes/datetimes.py

```diff
@@ -634,6 +636,49 @@ def strftime(self, date_format: str) -> Index:
         """
         return Index(self.to_series().dt.strftime(date_format))
 
+    def indexer_at_time(self, time: Union[datetime.time, str], asof: bool = False) -> Index:
+        """
+        Return index locations of values at particular time of day
+        (e.g. 9:30AM).
+
+        Parameters
+        ----------
+        time : datetime.time or str
+            Time passed in either as object (datetime.time) or as string in
+            appropriate format ("%H:%M", "%H%M", "%I:%M%p", "%I%M%p",
+            "%H:%M:%S", "%H%M%S", "%I:%M:%S%p", "%I%M%S%p").
+
+        Returns
+        -------
+        values_at_time : Index of integers
+
+        Examples
+        --------
+        >>> kidx = ks.date_range("2000-01-01", periods=3, freq="T")
+        >>> kidx # doctest: +NORMALIZE_WHITESPACE
```

Comment:
nit: 2 spaces between `kidx` and `# doctest`.

Reply:
Updated.